_observing-your-data/alerting/api.md

```diff
@@ -1342,6 +1342,19 @@ Introduced 1.0
 
 Returns an array of all alerts.
 
+**Query Parameters**
```

Comment:
Might be a good idea to have some examples of how these path parameters are used in the request. See [Path and HTTP methods](https://github.com/opensearch-project/documentation-website/blob/main/API_STYLE_GUIDE.md#path-and-http-methods) in API_STYLE_GUIDE.md.